packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -644,6 +644,7 @@ export class SigningCosmWasmClient extends CosmWasmClient {
     fee: StdFee,
     memo: string,
     explicitSignerData?: SignerData,
+    timeoutHeight?: Long,
```

Comment:
Can you make that a bigint? CosmJS 0.32 will pretty much get rid of long (see #1484)